tool/create-exercise

```diff
@@ -226,38 +242,32 @@ Future main(args) async {
   await new File("${exerciseDir.path}/pubspec.yaml").writeAsString(pubTemplate(name));
 
   if (arguments["spec-path"] != null) {
-    // The output from test generation is not always well-formatted, use dartfmt to clean it up
-    final res = await Process.run("dartfmt", ["-l", "120", "-w", testFileName], runInShell: true);
-    if (!res.stdout.toString().isEmpty) {
-      stdout.write(res.stdout);
-    }
-
-    if (!res.stderr.toString().isEmpty) {
-      stderr.write(res.stderr);
+    // Generate README
+    final dartRoot = "${dirname(Platform.script.toFilePath())}/..";
+    final configletLoc = "$dartRoot/bin/configlet";
+    final genSuccess = await runProcess(
+        configletLoc, ["generate", "$dartRoot", "--spec-path", arguments["spec-path"], "--only", name]);
+    if (genSuccess) {
+      stdout.write("Successfully created README.md\n");
+    } else {
+      stderr.write("Warning: `configlet generate` exited with an error, 'README.md' is likely malformed.\n");
     }
 
-    if (res.exitCode != 0) {
-      stderr.write("Warning: dartfmt exited with a code of ${res.exitCode}, '$testFileName' is likely malformed.\n");
-    } else {
+    // The output from test generation is not always well-formatted, use dartfmt to clean it up
+    final fmtSuccess = runProcess("dartfmt", ["-l", "120", "-w", testFileName]);
```

Comment:
Recommend not using testFileName, but the path to the new exercise ie : "$dartRoot/exercises/pangram"

Reply:
Makes sense!